cmd/builder/internal/builder/config.go

```diff
@@ -207,34 +203,9 @@ func (c *Config) ParseModules() error {
 		return err
 	}
 
-	if c.Providers != nil {
-		providers, err := parseModules(*c.Providers)
-		if err != nil {
-			return err
-		}
-		c.Providers = &providers
-	} else {
```

Comment:
is it a problem if no providers are defined, should the builder error out in that case?